docs/system/system-metrics.md

```diff
@@ -11,6 +11,11 @@ metrics in OpenTelemetry. Consider the [general metric semantic
 conventions](/docs/general/metrics.md#general-metric-semantic-conventions) when creating
 instruments not explicitly defined in the specification.
 
+Note that `system.*` namespace should only be used for hosts' metrics.
+For resource attributes related to a host the `host.*` namespace should be used.
+The `system.*` namespace for metrics should be used only when the metrics are collected from within the target system.
+For metrics collected centrally which are related to known entities like containers the respective namespace should be used.
```

Comment:
What does "collected centrally" here means? 🤔 Does this mean like:

> For metrics collected directly from known entities (e.g., containers), the metrics should be reported in the entity's respective namespace (`container.*`).

Reply:
Ah I see. I think then the text would be clearer if we mention this somehow like:

> Metrics collected from technology-specific, well-defined APIs (e.g. Kubelet's API, container runtimes?)
> should be reported under their respective namespace (e.g. `k8s.`, `container.*`)

Maybe? It's just a draft, I didn't check if we do have those namespaces or if they make sense, just as an idea for you.

Reply:
That makes sense! Thanks for the suggestions!
I also verify that on collector's side the metrics follow this approach:
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/documentation.md
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/documentation.md